gr4vy-iOS/Models/Gr4vyBuyer.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  Gr4vyBuyer.swift
+//  gr4vy-ios
+//
+
+import Foundation
+
+public struct Gr4vyBuyer: Codable {
+    let displayName: String?
+    let externalIdentifier: String?
+    let billingDetails: Gr4vyBillingShippingDetails?
+    let shippingDetails: Gr4vyBillingShippingDetails?
+    
+    public init(
+        displayName: String? = nil,
+        externalIdentifier: String? = nil,
+        billingDetails: Gr4vyBillingShippingDetails? = nil,
+        shippingDetails: Gr4vyBillingShippingDetails? = nil
+    ) {
+        self.displayName = displayName
+        self.externalIdentifier = externalIdentifier
+        self.billingDetails = billingDetails
+        self.shippingDetails = shippingDetails
+    }
+}
+
+public struct Gr4vyBillingShippingDetails: Codable {
```

Comment:
Combining the two `billingDetails` and `shippingDetails` with the same properties

Reply:
As in https://github.com/gr4vy/gr4vy-android/pull/14, maybe these should be different as the shipping details don't have the `taxId`?

Reply:
updated